Source/monster.cpp

```diff
@@ -3469,92 +3409,54 @@ BOOL __fastcall M_CallWalk(int i, int md)
 	return 0;
 }
 
-bool __fastcall M_PathWalk(int i)
+BOOL __fastcall M_PathWalk(int i)
 {
-	int v1; // esi
+	char path[25];
 	BOOL(__fastcall * Check)
-	(int, int, int); // ecx
-	char path[25];   // [esp+4h] [ebp-1Ch]
+	(int, int, int);

```

Comment:
Put (int, int, int) on the same line as the Check.

Reply:
We need to adjust .clang-format for that to be possible.